### PR TITLE
Feature/teamwrapper

### PR DESCRIPTION
**Pull request by marek-palenik**

### CODE REVIEW

File: src/components/teamstats/TeamStatsTable.module.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.teamStatsTable {
+    color: #6d6d6d;
+}
+
+.teamStatsTable > div > b {
```
Comment (lukasorgovan) [inline src/components/teamstats/TeamStatsTable.module.css]: too much specific rule, you should add a class to the "b" element if necessary.  also float is not a good solution. The wrapper should have property `display: flex` and `justify-content: space-between`. Float could potentially break the design in this case.

File: src/components/teamstats/TeamStatsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (lukasorgovan) [inline src/components/teamstats/TeamStatsTable.tsx]: teamstatsTABLE is quite bad name , "tile" as would be more descriptive.

File: src/components/teamstats/TeamStatsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+export interface TeamStats {
+    commitsPerTeam: Number;
+    jiraTickets: Number;
+}
+
+const TeamStatsTable = (stats: TeamStats) => {
+    return (
+        <div className={styles.teamStatsTable}>
+            <div>Commits per team <b>{stats.commitsPerTeam}</b></div>
```
Comment (lukasorgovan) [inline src/components/teamstats/TeamStatsTable.tsx]: those labels could be also passed as props, ideally they would be "statsRow" component where you would have label prop and stat prop.

File: src/components/teamwrapper/TeamWrapper.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import TeamStatsTable, {TeamStats} from "../teamstats/TeamStatsTable";
+
+export interface Team {
+    name: String;
+    detailUrl:String
+}
+
+const TeamWrapper = (team: Team & TeamStats) => {
+    return (
+        <a className={styles.teamwrapper} href={`/todo/route/${team.detailUrl}`}>
```
Comment (lukasorgovan) [inline src/components/teamwrapper/TeamWrapper.tsx]: a react router should be used (there is a Link component in react router... https://reacttraining.com/react-router/web/example/basic

**Comment by marek-palenik** [comment]
closing, due bad merge/rebase git commits

**Pull request closed without merging**